packages/phone-number-privacy/signer/README.md

```diff
@@ -1,3 +1,30 @@
-## Phone Number Privacy
+# Phone Number Privacy - Signer Service
 
-A firebase function that generates unique salts for phone numbers.
+A service run by distributed partners that generates unique partial signatures for phone number salts. Using a threshold BLS signature scheme, when K/N signatures are combined, a deterministic signature is obtained.
+
+## Configuration
+
+You can use the following environment variables to configure the Phone Number Privacy - Signer service:
+
+- `NODE_ENV` - eg. development, production
+- `SERVER_PORT` - The port on which the express node app runs.
+- `BLOCKCHAIN_PROVIDER` - The blockchain node provider for on chain requests. This could be a node with RPC set up.  For development testing you can use https://alfajores-forno.celo-testnet.org
+- `DB_HOST` - The URL under which your database is accessible.  Supports Postgres, MSSQL, MySQL, MariaDB, SQLite3, Oracle.
+- `DB_USERNAME` - DB configuration: The DB username.
+- `DB_PASSWORD` - DB configuration: The DB password.
+- `DB_DATABASE` DB configuration: The DB database name.
+
+Currently, the service only supports a key maintained in Azure Key Vault (AKV).  You can set the following environment variables to configure the AKV connection.
+
+- `KEYVAULT_AZURE_CLIENT_ID` - Azure KeyVault configuration: Client id.
+- `KEYVAULT_AZURE_CLIENT_SECRET` - Azure KeyVault configuration: Client secret.
```

Comment:
These need to be from a service principal account that they create. We can link instructions here: